mathics/builtin/makeboxes.py

```diff
@@ -485,7 +485,7 @@ def format_operator(operator) -> Union[String, BaseElement]:
                 return op
             return operator
 
-        precedence = prec.value
+        precedence = prec.value if hasattr(prec, "value") else 0
```

Comment:
Value has been removed from Symbol, so is a workaround #691 I think handles this better, but other than fixing things so that we don't call this routine when it shouldn't be called I don't recall the details

@mmatera I think you had the key insight here.

Reply:
@rocky, maybe it would be better to set the default to 670 (which is the default for precedence).

Reply:
0 means no parenthesis ever.  670 adds a parenthesis for everything except Get `<<`. 

So restating are you sure we want parenthesis most of the time?

Reply:
OK, in that case, let's keep 0...

Reply:
In another PR we may add a constant named "NEVER_ADD_PARENTHESIS".  There was more refactoring oef the parenthesize function going on in #691